### PR TITLE
Updated ubi-minimal, go-toolset, go-toolset, ubi and ose images

### PR DESCRIPTION
**Pull request by hbelmiro**
## The issue resolved by this Pull Request:
Resolves https://github.com/opendatahub-io/data-science-pipelines-operator/issues/392

## Description of your changes:
This PR updates the ubi-minimal, go-toolset, go-toolset, ubi and ose container images.

## Checklist
- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19.10 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19.13 as builder
```
Comment (zdtsw) [inline Dockerfile:2]:
```suggestion
FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
```

File: kfdef/kfdef.yaml [hunk 2]
```diff
@@ -15,15 +15,15 @@ spec:
...
           - name: IMAGES_DSPO
             value: quay.io/opendatahub/data-science-pipelines-operator:main
           - name: IMAGES_MOVERESULTSIMAGE
-            value: registry.access.redhat.com/ubi8/ubi-micro:8.7
+            value: registry.access.redhat.com/ubi8/ubi-micro:8.8
           - name: IMAGES_MARIADB
             value: registry.redhat.io/rhel8/mariadb-103:1-188
           - name: IMAGES_OAUTHPROXY
-            value: registry.redhat.io/openshift4/ose-oauth-proxy:v4.12.0
+            value: registry.redhat.io/openshift4/ose-oauth-proxy:v4.13.0
```
Comment (zdtsw) [inline kfdef/kfdef.yaml:26]: is oauth-proxy also impacted by this?

**Review by zdtsw — COMMENTED** [review]
1.19 is equivalent to the latest
so you do not need to use the current latest one 1.19.13 and keep updating

File: kfdef/kfdef.yaml (hunk 2 above)
Comment (hbelmiro) [reply kfdef/kfdef.yaml:26]: Not really. I saw ose-cli and imagined that ose-auth-proxy should be updated as well.

Comment (hbelmiro) [reply kfdef/kfdef.yaml]: Reverted. Thank you.

**Pull request merged**